www/src/views/starter-library/starter-list.js

```diff
@@ -171,6 +170,7 @@ const StartersList = ({ urlState, starters, count, sortRecent }) => {
       </div>
     )
   }
+  return <React.Fragment />
```

Comment:
not sure here if proper - there are `if`s check before that so I think we need to return react element here?

Reply:
I'd maybe tweak this, e.g.

```jsx
if (!count) {
  return null; // or empty state from above?
}

return (

); // regular flow 
```

Reply:
i think we could just return null

Reply:
changed to `null`